pkg/corerp/api/v20231001preview/environment_conversion_test.go

```diff
@@ -291,7 +291,11 @@ func TestConvertVersionedToDataModel(t *testing.T) {
 		},
 		{
 			filename: "environmentresource-terraformrecipe-localpath.json",
-			err:      &v1.ErrClientRP{Code: v1.CodeInvalid, Message: fmt.Sprintf(invalidLocalModulePathFmt, "../not-allowed/")},
+			err:      &v1.ErrClientRP{Code: v1.CodeInvalid, Message: fmt.Sprintf(invalidModulePathFmt, "../not-allowed/")},
+		},
+		{
+			filename: "environmentresource-terraformrecipe-unsupported.json",
```

Comment:
This will be denied by the `ParseRequestURI` and the other one by the `ParseModuleSource`, right?

Reply:
They should both error out since neither of these are a TF registry or an HTTP URL